docker-compose.Example.yml

```diff
@@ -1,7 +1,8 @@
 services:
   server:
-    command: --debug --dbaddr db --dbuser pokerogue --dbpass pokerogue --dbname pokeroguedb --gameurl http://localhost:8000 --callbackurl http://localhost:8001 
-    image: ghcr.io/pagefaultgames/rogueserver:master
+    env_file:
+      - beta.env
+    image: rogueserver:latest
```

Comment:
Idk if you want to replace the image here.

Reply:
@flx-sta Sorry, something weird is happening with github and I didn't see an email about this. This was changed locally and pushed so walker could see my code and why things weren't working, but will be fixed and reverted before it comes out of draft (hopefully :) ). I've got a note to do it, so hopefully I'll actually remember to do it